stable/sonarqube/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: sonarqube
 description: Sonarqube is an open sourced code quality scanning tool
-version: 1.0.4
+version: 1.0.5
```

Comment:
Please bump the minor version as this adds a new feature

Reply:
Ok will do so.